Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/ParticleSelector.hpp

```diff
@@ -62,9 +62,10 @@ class ParticleSelector final : public IAlgorithm {
     bool removeCharged = false;
     /// Remove neutral particles.
     bool removeNeutral = false;
-
     /// Remove secondaries.
     bool removeSecondaries = false;
+    /// Exclude particles depending on absolute pdg value
+    std::vector<int> excludeAbsPdgs;
```

Comment:
alternatively this could be a set

Reply:
true, probably this is also natively supported by pybind11, right?

Reply:
I would think so yes

Reply:
Good idea, but I would keep it as it is for now.